arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -1793,6 +1790,8 @@
 		F140E1EC23F35EA6008AC44E /* DisplayAnnotation.storyboard */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; path = DisplayAnnotation.storyboard; sourceTree = "<group>"; };
 		F142F769267C12E400BD608A /* SanFrancisco.mmpk */ = {isa = PBXFileReference; lastKnownFileType = file; path = SanFrancisco.mmpk; sourceTree = "<group>"; };
 		F142F76D267C12F800BD608A /* Yellowstone.mmpk */ = {isa = PBXFileReference; lastKnownFileType = file; path = Yellowstone.mmpk; sourceTree = "<group>"; };
+		F142F79D267D755700BD608A /* SanFrancisco.tpkx */ = {isa = PBXFileReference; lastKnownFileType = file; name = SanFrancisco.tpkx; path = ../../../Downloads/SanFrancisco.tpkx; sourceTree = "<group>"; };
+		F142F79E267D755700BD608A /* streetmap_SD.tpkx */ = {isa = PBXFileReference; lastKnownFileType = file; name = streetmap_SD.tpkx; path = ../../../Downloads/streetmap_SD.tpkx; sourceTree = "<group>"; };
```

Comment:
![image](https://user-images.githubusercontent.com/9660181/122830726-9bc4da80-d29d-11eb-9fe3-65cc5c6479bb.png)


The file paths here aren't correct. They should be relative paths in the `Portal Data` folder. Maybe you've dragged these files into the file navigator from `Downloads` folder? 😏

Reply:
Hmm I see, please try again!